test/units/TEST-13-NSPAWN.nspawn.sh

```diff
@@ -1129,7 +1129,7 @@ testcase_unpriv() {
 
     local tmpdir name
     tmpdir="$(mktemp -d /var/tmp/TEST-13-NSPAWN.unpriv.XXX)"
-    name="unpriv-${tmpdir##*.}"
+    name="unprv-${tmpdir##*.}"
```

Comment:
Why is this changed?

Reply:
to make it shorter. the userns is named after that. #35685 has a patch that removes the userns naming limit, but that's another story...